lib/wda/iproxy.js

```diff
@@ -1,98 +1,33 @@
-import log from '../logger';
-import { killProcess } from './utils';
-import B from 'bluebird';
-import { logger } from 'appium-support';
-import { SubProcess } from 'teen_process';
-import { checkPortStatus } from 'portscanner';
-import { waitForCondition } from 'asyncbox';
-
-
-const IPROXY_STARTUP_TIMEOUT = 5000;
-
-const iproxyLog = logger.getLogger('iProxy');
+import { utilities } from 'appium-ios-device';
+import net from 'net';
 
 class iProxy {
-  constructor (udid, localport, deviceport, detached = true) {
-    this.expectIProxyErrors = true;
```

Comment:
we should also update the client code which uses `detached` and `expectIProxyErrors`

Reply:
Done. They are both removed